### PR TITLE
jbig: fix iOS build + delete fPIC if shared

### PR DESCRIPTION
**Pull request by SpaceIm**
Specify library name and version:  **lib/1.0**

This is also a good place to share with all of us **why you are submitting this PR** (specially if it is a new addition to ConanCenter): is it a dependency of other libraries you want to package? Are you the author of the library? Thanks!

---

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by prince-chrismc — APPROVED** [review]

**Comment by SpaceIm** [comment]
Did I break CI again? 😗

**Comment by prince-chrismc** [comment]
Lots of PRs are stuck today....

**Review by SSE4 — APPROVED** [review]

**Comment by jgsogo** [comment]
Can you, please, skip `test_package` if cross-building? https://github.com/conan-io/conan-center-index/blob/85309f29244dfda31bbaef7926674a2ca6e115db/recipes/jbig/all/test_package/conanfile.py#L14

**Review by prince-chrismc — APPROVED** [review]

**Review by jgsogo — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

File: recipes/jbig/all/CMakeLists.txt [hunk 1]
```diff
@@ -8,35 +8,27 @@ if(MSVC)
...
+        LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+        ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
 
-set(OTHER_TARGETS)
+install(FILES source_subfolder/libjbig/jbig.h DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
 
 if (BUILD_EXECUTABLES)
     add_executable(pbmtojbg source_subfolder/pbmtools/pbmtojbg.c)
-    target_link_libraries(pbmtojbg ${CMAKE_PROJECT_NAME})
+    target_link_libraries(pbmtojbg ${PROJECT_NAME})
```
Comment (SSE4) [inline recipes/jbig/all/CMakeLists.txt:26]: I think both `PROJECT_NAME` and `CMAKE_PROJECT_NAME` have the same effect

Comment (SpaceIm) [reply recipes/jbig/all/CMakeLists.txt:26]: In this context, it's the same value yes. CMAKE_PROJECT_NAME refers to the top level project name, while PROJECT_NAME is the name of the most recent called project().

**Comment by SpaceIm** [comment]
@jgsogo the last revision of jbig in conan-center has not been updated with this PR.

**Comment by jgsogo** [comment]
Hi, @SpaceIm , which remote URL are you using? We have stopped uploading packages to `https://conan.bintray.com` and now only the new default remote `https://center.conan.io` gets the new revisions: https://blog.conan.io/2021/06/10/New-conan-release-1-37.html

**Comment by SpaceIm** [comment]
Completly missed that, thanks !

**Pull request merged**